src/commcare_cloud/ansible/roles/common_installs/tasks/main.yml

```diff
@@ -11,25 +11,13 @@
   apt_repository: repo='deb http://apt.postgresql.org/pub/repos/apt/ {{ ansible_lsb.codename }}-pgdg main' state=present
   register: add_postgres_repo
 
-- name: Add pango repo for trusty
```

Comment:
Does `libpango1.0-0` still get installed without this? And what version?

Reply:
```
# dpkg -l| grep libpango1
ii  libpango1.0-0:amd64                   1.40.14-1ubuntu0.1                amd64        Layout and rendering of internationalized text (transitional package)
# lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.2 LTS
Release:	18.04
Codename:	bionic
```